lib/stimulus_reflex/test_case.rb

```diff
@@ -25,9 +25,10 @@ module Behavior
     def build_reflex(opts = {})
       channel = opts.fetch(:channel, TestChannel.new(opts.fetch(:connection, {})))
       element = opts.fetch(:element, StimulusReflex::Element.new)
+      version = StimulusReflex::VERSION
 
       self.class.reflex_class.new(
-        channel, element: element, url: opts.fetch(:url, ""), method_name: method_name_from_opts(opts), params: opts.fetch(:params, {})
+        channel, element: element, url: opts.fetch(:url, ""), method_name: method_name_from_opts(opts), params: opts.fetch(:params, {}), version: version
```

Comment:
It looks like `version` is part of `client_attributes`. Maybe the client attributes key is newer? To get this running on our existing test suite I had to wrap it like: `client_attributes: {version: version}`.

Reply:
Interesting, not sure how I missed that! I can confirm though that I get the same error message. I never thought to actually _use_ my version of the gem in our app to verify that it worked 🤦 

I'll update this PR with your suggestion and see if I can run our specs successfully. Thanks!

Reply:
@jasoncharnes I confirmed that adding the `client_attributes` key also worked for me! 

The only other issue I can see is, and correct me if I'm wrong, but this PR as-is would force anyone using SR `3.5.0.pre8` and below to update to SR `3.5.0.pre9` to continue using `stimulus_reflex_testing`. Otherwise, they'd see the same error you have above, I think.

I think I could address that by creating two separate `Behavior` modules, one for the behavior in SR `3.5.0.pre8` and below, and one for later. Do you think I should add that?

Reply:
> The only other issue I can see is, and correct me if I'm wrong, but this PR as-is would force anyone using SR 3.5.0.pre8 and below to update to SR 3.5.0.pre9 to continue using stimulus_reflex_testing. Otherwise, they'd see the same error you have above, I think.

I wondered about that, too.

> I think I could address that by creating two separate Behavior modules, one for the behavior in SR 3.5.0.pre8 and below, and one for later. Do you think I should add that?

Is `StimulusReflex::VERSION` available in earlier versions?

The separate modules would work! I would also be okay with storing the arguments to `self.class.reflex_class.new` in a variable that has a different value based on the version. Whatever you find best! 🙂

Reply:
> Is `StimulusReflex::VERSION` available in earlier versions?

Yep! It's available as far back as version 0.1.0.

> The separate modules would work! I would also be okay with storing the arguments to `self.class.reflex_class.new` in a variable that has a different value based on the version. Whatever you find best! 🙂

Ok, cool. The latter might be the simpler way to go about it since there would only be two branches at this point.

Reply:
Sounds good! Feel free to ping me when it's ready and I'll give it a review. Thanks for all your help!

Reply:
Looks good, thanks again for you help!